app/controllers/report_controller/reports.rb

```diff
@@ -391,7 +391,7 @@ def check_tabs
       else
         if @edit[:new][:fields].length == 0
           add_flash(_("Preview tab is not available until at least 1 field has been selected"), :error)
-        elsif @edit[:new][:model] == "Chargeback" && !valid_chargeback_fields
+        elsif Chargeback.subclasses.collect(&:to_s).include?(@edit[:new][:model]) && !valid_chargeback_fields
```

Comment:
Same comment here as in https://github.com/ManageIQ/manageiq/pull/8989/files. Maybe you can do it in a followup PR.